EIPS/eip-5313.md

```diff
@@ -0,0 +1,58 @@
+---
+eip: 5313
+title: Light Contract Ownership
+description: An interface for identifying ownership of contracts
+author: William Entriken (@fulldecent)
+discussions-to: https://ethereum-magicians.org/t/eip-5313-light-contract-ownership/10052
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-07-22
+---
+
+## Abstract
+
+This specification defines the minimum interface required to identify an account that controls a contract.
+
+## Motivation
+
+This is a slimmed-down alternative to [EIP-173](./eip-173).
+
+## Specification
+
+The key word “MUST” in this document is to be interpreted as described in RFC 2119.
+
+Every contract compliant with this EIP MUST implement the `EIP5313` interface.
+
+```solidity
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.15;
+
+/// @title EIP-5313 Light Contract Ownership Standard
+interface EIP5313 {
+    /// @notice Get the address of the owner    
+    /// @return The address of the owner
+    function owner() view external returns(address);
+}
+```
+
+## Rationale
+
+Key factors influencing the standard: 
+
+- Minimize the number of functions in the interface
+- Backwards compatibility with existing contracts
+
+This standard can be (and has been) extended by other standards to add additional ownership functionality. The smaller scope of this specification allows more and more straightforward ownership implementations, see limitations explained in EIP-173 under "other schemes that were considered".
+
+## Backwards Compatibility
+
+Every contract that implement EIP-173 already implements this specification.
+
+## Security Considerations
+
+Because this specification does not require EIP-165 (and even if it did, due to limits of ERC-165 especially with single-function interfaces) interrogating with EIP-5313 cannot result in complete certainty that the result is the owner. For example, another function with the same function signature may return some value that is interpreted by an EIP-5313 interrogator to be the owner. If an interrogator is only using EIP-5313 to identify if a claimant is the owner, then the impact of this risk is minimized. But if the interrogator is, for example, sending a valuable NFT to the EIP-5313 identified ownwer of any contract on the network, then the risk is heightened.
```

Comment:
```suggestion
Because this specification does not extend EIP-165, calling this EIP's `owner` function cannot result in complete certainty that the result is indeed the owner. For example, another function with the same function signature may return some value that is then interpreted to be the true owner. If this EIP is used solely to identify if an account is the owner of a contract, then the impact of this risk is minimized. But if the interrogator is, for example, sending a valuable NFT to the identified owner of any contract on the network, then the risk is heightened.
```